src/components/Backdrop/Backdrop.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+import classes from './Backdrop.module.css';
+const backdrop = (props) => (props.show ? <div className={classes.Backdrop} onClick={props.clicked} /> : null);
```

Comment:
Your efforts are appreciated.. but as per the UI library we are using there is Modal component we can use that provide us the functionality to use Modal with proper backdrop so try to use it instead as it is ready made